packages/wallets/coinbase/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@solana/wallet-adapter-coinbase",
+    "version": "0.1.3",
+    "author": "Solana Maintainers <maintainers@solana.foundation>",
+    "repository": "https://github.com/solana-labs/wallet-adapter",
+    "license": "Apache-2.0",
+    "type": "module",
+    "sideEffects": false,
+    "main": "lib/cjs/index.js",
+    "module": "lib/esm/index.mjs",
+    "types": "lib/types/index.d.ts",
+    "exports": {
+        "import": "./lib/esm/index.mjs",
+        "require": "./lib/cjs/index.js"
+    },
+    "files": [
+        "lib",
+        "src",
+        "LICENSE"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "clean": "shx rm -rf lib/*",
+        "build": "yarn clean && tsc -p tsconfig.json && tsc-esm -p tsconfig.json && tsc -p tsconfig.cjs.json",
+        "postbuild": "echo '{\"type\":\"commonjs\"}' | npx json > lib/cjs/package.json && echo '{\"type\":\"module\"} ' | npx json > lib/esm/package.json"
+    },
+    "dependencies": {
+        "@solana/wallet-adapter-base": "^0.9.4",
+        "@solana/web3.js": "^1.36.0"
+    }
```

Comment:
Can you copy the peer/dev/dep structure here?

https://github.com/solana-labs/wallet-adapter/blob/244af02cb3dcf9334abf61cb9a11e497bde86d57/packages/wallets/exodus/package.json#L28-L36